README

```diff
@@ -105,7 +105,7 @@ and edit the "local.conf" file. Here is an example:
 
 vim conf/local.conf
 [...]
-MACHINE ??= "sama5d3-xplained"
+MACHINE ??= "sama5d2-xplained"
```

Comment:
Well, this is just an example of one machine. As everybody don't use the same one, I keep the sama5d3 here...

Reply:
Ok, it's for coherency: sorry for not having read the commit message first: makes sense, I take it! ;-)